src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -1,42 +1,45 @@
-import { AppAuthStrategyOptions } from '@/types';
+import {
+  AppAuthStrategyOptions,
+  GitHubIssuesSomeoneElseCaresAbout,
+} from '@/types';
 
 export class GitHubApp {
+  num: number;
   org: string;
   auth: AppAuthStrategyOptions;
-  project;
+  project: GitHubIssuesSomeoneElseCaresAbout;
 
-  constructor(org: string, opts) {
-    this.org = org;
-    this.auth = {
-      appId: Number(opts.GH_APP_IDENTIFIER),
-      privateKey: opts.GH_APP_SECRET_KEY.replace(/\\n/g, '\n'),
-    };
-    this.project = {
-      node_id: opts.ISSUES_PROJECT_NODE_ID || 'PVT_kwDOABVQ184AOGW8',
-      product_area_field_id:
-        opts.PRODUCT_AREA_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgJEBno',
-      status_field_id: opts.STATUS_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgI_7g0',
-      response_due_date_field_id:
-        opts.RESPONSE_DUE_DATE_FIELD_ID || 'PVTF_lADOABVQ184AOGW8zgLLxGg',
-    };
+  constructor(obj) {
+    this.num = obj.num;
+    this.org = obj.org;
+    this.auth = obj.auth;
+    this.project = obj.project;
   }
 }
 
-export class GitHubAppsRegistry {
-  apps: Map<string, GitHubApp>;
+class GitHubAppsConfigHelper {
+  configs: Map<number, any>; // much looser typing than apps
 
   constructor() {
-    this.apps = new Map<string, GitHubApp>();
+    this.configs = new Map<number, object>();
   }
 
-  register(app) {
-    this.apps.set(app.org, app);
+  forNumber(num: number): object | undefined {
+    if (!this.configs.has(num)) {
+      this.configs.set(num, { num: num });
+    }
+    return this.configs.get(num);
   }
+}
 
-  pop(org) {
-    const app = this.load(org);
-    this.apps.delete(org);
-    return app;
+export class GitHubAppsRegistry {
+  apps: Map<string, GitHubApp>;
+
+  constructor(configs) {
+    this.apps = new Map<string, GitHubApp>();
+    for (const config of configs.configs.values()) {
```

Comment:
the `configs.configs.values()` here was unclear to me what it was, configs here is more accurately `GitHubAppsRegistry`? Can this be renamed to be `githubAppsRegistry` or something similar?

Reply:
No, it's `GitHubAppsConfigHelper`. Renamed to `configHelper` in 219157b307014f98e6715aa4de02615d656a84be (along with other changes :)).